src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,45 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkAlive() {
+    if (this.health <= 0) {
+      const index = Animal.alive.indexOf(this);
+
+      if (index !== -1) {
+        Animal.alive.splice(index, 1);
+      }
+    }
```

Comment:
The `checkAlive` method is correctly checking if the health is less than or equal to zero and removing the animal from the `Animal.alive` array. Ensure that this method is called appropriately in the context where an animal's health might change.